src/main/java/io/github/microcks/testcontainers/model/DailyInvocationStatistic.java

```diff
@@ -0,0 +1,263 @@
+/*
+ * Microcks API v1.10
+ * API offered by Microcks, the Kubernetes native tool for API and microservices mocking and testing (microcks.io)
+ *
+ * The version of the OpenAPI document: 1.10.0
+ * Contact: laurent@microcks.io
+ *
+ * NOTE: This class comes from microcks-java-client project : https://github.com/microcks/microcks-java-client
+ */
```

Comment:
Could you place here the standard header like in https://github.com/microcks/microcks-testcontainers-java/blob/main/src/main/java/io/github/microcks/testcontainers/MicrocksContainer.java#L1-L15 ?

Reply:
Done